client/landing/jetpack-cloud/components/scan-history-item/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Badge from 'components/badge';
+import LogItem from '../log-item';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ScanHistoryItem extends Component {
+	static propTypes = {
+		object: PropTypes.object,
+	};
+
+	entryActionClassNames( entry ) {
+		return {
+			'is-fixed': entry.action === 'fixed',
+			'is-ignored': entry.action === 'ignored',
+		};
+	}
+
+	renderEntryHeader( entry ) {
+		// React complains because we shouldn't wrap a <div> tag inside <p> tag. This happens
+		// because LogItem surrounds its content by a <p> tag, and the Badge is a <div> tag.
+		// Should we create our own Badge component?
+		return (
+			<>
+				<div className="scan-history-item__subheader">
+					<span className="scan-history-item__date">Threat found on { entry.detectionDate }</span>
```

Comment:
The string here needs to be passed through `localize`. See [here](https://github.com/Automattic/wp-calypso/tree/master/packages/i18n-calypso#translate-method) for detailed usage.

Reply:
Done!